frontend/src/app/offers/create-offer/create-offer.component.html

```diff
@@ -0,0 +1,304 @@
+<div *ngIf="isAdmin; else hasOrg">
+  <p>Administrator nie może tworzyć nowych ofert.</p>
+</div>
+<ng-template #hasOrg>
+  <ng-container *ngIf="!hasOrganization; else offerForm">
+  <p>Aby dodać ofertę musisz posiadać organizację.
+    <a href="{{ djangoRoot }}/organizations/create">Możesz ją założyć tutaj</a>
+  </p>
+  </ng-container>
+</ng-template>
+<ng-template #offerForm>
+<h2>{{ inEditMode ? "Edycja" : "Tworzenie" }} oferty:</h2>
+<form (ngSubmit)="onSubmit(offer)" [formGroup]="form">
+  <div class="form-group row">
+    <label for="title" class="col-sm-3 col-form-label">Tytuł oferty:</label>
+    <div class="col-sm-6">
+      <input 
+        type="text"
+        class="form-control"
+        placeholder="Tytuł oferty..." 
+        id="title"
+        formControlName ="title"
+        required>
+      <span class="form-text text-muted" 
+      *ngIf="!form.get('title').valid && form.get('title').touched">
```

Comment:
We get this line a lot: !form.get('title').valid && form.get('title').touched", only the name of the input changes in other lines. We can write one function to solve this logic.
You can define a method inside a component:
```
isFormInputInvalid(inputStringId: string): boolean {
    const input = this.form.get(inputStringId);
    return !input.valid && input.touched;
  }
```
and you use it in a template like:
`
*ngIf="isFormInputInvalid('title')">
`